homework-submission/myapp/src/stores/todoStore.js

```diff
@@ -0,0 +1,49 @@
+import { observable, action } from "mobx";
+import Data from "./../data/data";
+
```

Comment:
Oh! you missed a thing in your homework 👌
```suggestion
configure({ enforceActions: 'observed' })
```

You don't have to put it there but for a development purposes it's preferred to be here always while doing it.